dev/make/compiler_definitions/dpcpp.mk

```diff
@@ -28,7 +28,7 @@ CORE.SERV.COMPILER.dpcpp = generic
 -Zl.dpcpp =
 -DEBC.dpcpp = -g
 
-COMPILER.lnx.dpcpp = icpx -fsycl -m64 -stdlib=libstdc++ -fgnu-runtime -fwrapv \
+COMPILER.lnx.dpcpp = icpx -fsycl -O0 -m64 -stdlib=libstdc++ -fgnu-runtime -fwrapv \
```

Comment:
what is this flag?

Reply:
Now, I am mostly experimenting with different compiler flags for optimization and try to found the best performance option, will deleted if there is no success

Reply:
any details on what the flag actually does? and how much impact does it have on perf?

Reply:
this is optimization level flag - fine for experiments but would have implications on perf and library
https://gcc.gnu.org/onlinedocs/gcc/Optimize-Options.html

Reply:
Seems like -O0 is default? Are perf results consistent or could differences be variable